src/inspector/node_protocol.pdl

```diff
@@ -37,3 +37,59 @@ experimental domain NodeTracing
   # Signals that tracing is stopped and there is no trace buffers pending flush, all data were
   # delivered via dataCollected events.
   event tracingComplete
+
+# Supports additional targets discovery and allows to attach to them.
+experimental domain NodeTarget
+
+  type TargetID extends string
+
+  # Unique identifier of attached debugging session.
+  type SessionID extends string
+
+  type TargetInfo extends object
+    properties
+      TargetID targetId
+      string type
+      string title
+      string url
+
+  # Sends protocol message over session with given id.
+  command sendMessageToTarget
```

Comment:
This does not exist in the new flattened world, sessionId attribute is passed as a top-level parameter instead. This allows us to not JSON-serialize and wrap all the commands that go into the nested targets such as workers.

Reply:
I can still see it here - https://paulirish.github.io/debugger-protocol-viewer/1-3/Target/

Reply:
We have not migrated all the use cases to flattened yet, it also is a public API method, so deprecation would beed to be graceful. Basically, it'll soon become deprecated and a year later it'll become noop.